tests/server/routes/cohorts.test.js

```diff
@@ -0,0 +1,18 @@
+const request = require('supertest')
+// const cheerio = require('cheerio')
+
+const server = require('../../server/routes')
+const db = require('../db/cohorts.test')
+
+beforeEach(() => {
+  db.reset()
+})
+
+test('GET / returns cohorts', () => {
+  request(server)
+    .get('/')
```

Comment:
You need to look into this as well. Isn't this test supposed to be for the cohorts route?